File: Easy/n3392.py
from typing import List 

def countSubarrays(self, nums: List[int]) -> int:
    res = 0
    n = len(nums)

    for i in range(n-2):
        f = nums[i]
        s = nums[i + 1]
        t = nums[i + 2]

        if (f + t) * 2 == s:
            res += 1

        return res

File: Easy/n3392.swift
func countSubarrays(_ nums: [Int]) -> Int {
    var res = 0
    let n = nums.count

    for i in 0..<n-2 {
        let f = nums[i]
        let s = nums[i + 1]
        let t = nums[i + 2]

        if (f + t) * 2 == s {
            res += 1
        }
    }

    return res
}